pages/citations.md

```diff
@@ -1,207 +0,0 @@
----
-layout: page-fullwidth
-title: "Publications about and by The Carpentries"
-permalink: /citations/
----
```

Comment:
Not sure I would completely kick this out, it's good to keep around to know how we might set such a section up

Reply:
Restored by 295ab288 with some placeholder text.